src/types.js

```diff
@@ -44,17 +44,17 @@ export const types = {
     container: true,
     defaultProps: {
       align: 'center',
-      justify: 'center',
+      justify: 'stretch',
```

Comment:
Having `center` as the default for both `align` and `justify` tends to be what you want for an initial experience using the designer. Once we've got more starter flavors of how components can be built up, I think those will handle the varied cases better. You can see this with the `header` Box starter.